### PR TITLE
migrateWarnFunc( jQuery, "isArray" moved to "if ( jQueryVersionSince(…

### PR DESCRIPTION
**Pull request by ildar-aim**
migrateWarnFunc( jQuery, "isArray" moved to the "if ( jQueryVersionSince( "3.3.0" ) ) {" condition https://github.com/jquery/jquery-migrate/issues/370

### CODE REVIEW

File: src/jquery/core.js [hunk 1]
```diff
@@ -106,6 +106,10 @@ if ( jQueryVersionSince( "3.2.0" ) ) {
 		return elem.nodeName && elem.nodeName.toLowerCase() === name.toLowerCase();
 	},
 	"jQuery.nodeName is deprecated" );
+
+        migrateWarnFunc( jQuery, "isArray", Array.isArray,
+                "jQuery.isArray is deprecated; use Array.isArray"
+        );
```
Comment (mgol) [inline src/jquery/core.js:112]: Please follow our code style which uses tabs, not spaces.

Comment (ildar-aim) [reply src/jquery/core.js]: done

**Comment by mgol** [comment]
@ildar-aim Please also sign our CLA as instructed above.

**Comment by ildar-aim** [comment]
> @ildar-aim Please also sign our CLA as instructed above.

CLA assistant says that I have signed the CLA for jquery/jquery-migrate

image


https://cla.js.foundation/jquery/jquery-migrate?pullRequest=373

**Comment by ildar-aim** [comment]
ok 
it seems I am not linked.

working on it...

**Comment by ildar-aim** [comment]
'Changing author info' https://docs.github.com/en/github/using-git/changing-author-info

totally has broken my PR.

I will try a new one.

**Pull request closed without merging**